download_script.py

```diff
@@ -0,0 +1,50 @@
+import requests
+import argparse
+
+# help function for better understanding of the scripts functionality
+
```

Comment:
No longer needed since it now precedes the rest of the script, just remove it.

```suggestion
# help function for better understanding of the scripts functionality

```